webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala

```diff
@@ -606,11 +556,12 @@ case class UserUpdatePasswordPayloadADM(requesterPassword: Password, newPassword
 object UserUpdatePasswordPayloadADM {
   def make(apiRequest: ChangeUserPasswordApiRequestADM): Validation[String, UserUpdatePasswordPayloadADM] = {
     val requesterPasswordValidation = apiRequest.requesterPassword
-      .map(Password.validationFrom)
+      .map(validateOneWithFrom(_, Password.from, a => a))
```

Comment:
```suggestion
      .map(validateOneWithFrom(_, Password.from, _))
```
Would this work?

Reply:
Why don't you try it out yourself?

Reply:
Sure I can check it, but I don't always checkout into the reviewing branch. After that seems `_` is not working, but `identity` works. Feel free to change it in all places in this PR if you feel it looks cleaner.

```suggestion
      .map(validateOneWithFrom(_, Password.from, identity))
```

Reply:
> but I don't always checkout into the reviewing branch. 

I agree that for performing a review this often is not necessary. However, it will greatly help you to clarify questions you might have which a compiler can easily answer for you.

As this is just a stylistic suggestion I take the liberty not to apply it.